src/_locales/en_US/messages.json

```diff
@@ -280,7 +280,7 @@
         }
     },
     "popup_info_youtube": {
-        "message": "YouTube not working? It's probably not because of Privacy Badger.",
+        "message": "YouTube not working? Ads playing? It's probably not because of Privacy Badger.",
```

Comment:
I think "ads playing?" followed by "it's not because of PB" is confusing. "It's not because of PB" makes sense together with "YT not working". On the subject of ads, PB doesn't try to block them in the first place; PB is beside the point when it comes to ads.

Also, we get more reports about ads than about videos not playing these days. 

So, maybe:
>Ads not blocked? YouTube not working? Visit [privacybadger.org](https://privacybadger.org/#Is-Privacy-Badger-breaking-YouTube) to learn more.

This is probably too verbose:
>Ads not blocked? Privacy Badger is not an ad blocker. YouTube not working? It's probably not because of Privacy Badger. Visit [privacybadger.org](https://privacybadger.org/#Is-Privacy-Badger-breaking-YouTube) to learn more.

Reply:
Makes sense. I'll use the first suggestion